Kconfig

```diff
@@ -11,6 +11,8 @@ mainmenu "Zeal 8-bit OS Configuration"
         string
         default "zeal8bit" if TARGET_ZEAL_8_BIT
         default "trs80" if TARGET_TRS_80
+        default "agon" if TARGET_AGON_LIGHT
+        default "example" if TARGET_EXAMPLE
```

Comment:
The example target has been removed from the latest HEAD, you can remove this

Reply:
Fixed